### PR TITLE
Adding reservoir level

### PR DESCRIPTION
**Pull request by ps2**
This is adding the code first submitted in #31

### CODE REVIEW

File: openomni/commands.py [hunk 1]
```diff
@@ -71,6 +71,7 @@ class PodStatusResponse(_BaseCommand):
 
     def Populate(self):
         self.minutes_active = ((ord(self.data[5]) & 0x0f) << 6) + (ord(self.data[6]) >> 2)
+        self.reservoir_level = round((float((((ord(self.data[6]) & 0x03) << 6) + (ord(self.data[7]) >> 2)))*50)/255, 0)
 
     def debug_detail(self):
```
Comment (panctronic) [inline openomni/commands.py]: Add this new data to debug_detail?

File: openomni/commands.py [hunk 2]
```diff
@@ -71,6 +71,7 @@ class PodStatusResponse(_BaseCommand):
 
     def Populate(self):
         self.minutes_active = ((ord(self.data[5]) & 0x0f) << 6) + (ord(self.data[6]) >> 2)
+        self.reservoir_level = round((float((((ord(self.data[6]) & 0x03) << 6) + (ord(self.data[7]) >> 2)))*50)/255, 0)
```
Comment (panctronic) [inline openomni/commands.py]: The last time this was here, it blew up on some 0x1d responses. It'd be good to add it in a try/except or something.

File: openomni/commands_test.py [hunk 3]
```diff
@@ -14,6 +14,12 @@ def test_parse_cancel(self):
         cmd, _ = commands.ParseCommand("1f05156b93e8620028".decode("hex"))
         self.assertTrue(isinstance(cmd, commands.CancelCommand))
 
+    def test_parse_1d_response(self):
+        resp, _ = commands.ParseCommand("1d18003e980000421fff81cf".decode("hex"))
+        self.assertTrue(isinstance(resp, commands.PodStatusResponse))
+        self.assertEqual(resp.minutes_active, 135)
+        self.assertEqual(resp.reservoir_level, 50.0)
```
Comment (panctronic) [inline openomni/commands_test.py]: 50 means >= 50 right?

File: openomni/commands.py (hunk 2 above)
Comment (ps2) [reply openomni/commands.py]: Do you know of any examples of messages that cause it to fail?

File: openomni/commands_test.py (hunk 3 above)
Comment (ps2) [reply openomni/commands_test.py]: Yes, iirc, 50 is the max.

**Pull request merged**